src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -120,13 +121,16 @@ export function IndexPatternProvider(Private, config, Promise, confirmModalPromi
 
     if (indexPattern.isUnsupportedTimePattern()) {
       if (!isUserAwareOfUnsupportedTimePattern(indexPattern)) {
-        const warning = (
-          'Support for time-intervals has been removed. ' +
-          `View the ["${indexPattern.title}" index pattern in management](` +
-          kbnUrl.getRouteHref(indexPattern, 'edit') +
-          ') for more information.'
-        );
-        notify.warning(warning, { lifetime: Infinity });
+        toastNotifications.addWarning({
+          title: 'Support for time-intervals has been removed',
+          text: (
+            <Fragment>
+              View the {' '}
+              <a href={kbnUrl.getRouteHref(indexPattern, 'edit')}>{indexPattern.title} index pattern in management</a>
+              {' '} for more information
+            </Fragment>
+          ),
+        });
       }
```

Comment:
Support for time intervals was removed

For more information, view the log* index pattern.

where "view the log* index pattern" is the link